modules/perception/camera/app/cipv_camera.cc

```diff
@@ -83,29 +83,37 @@ bool Cipv::DistanceFromPointToLineSegment(const Point2Df &point,
 bool Cipv::GetEgoLane(const std::vector<base::LaneLine> &lane_objects,
                       EgoLane *egolane_image, EgoLane *egolane_ground,
                       bool *b_left_valid, bool *b_right_valid) {
+  float x, y;
   for (size_t i = 0; i < lane_objects.size(); ++i) {
     const auto &lane_object = lane_objects[i];
     const size_t curve_image_point_size =
         lane_object.curve_image_point_set.size();
+    const size_t curve_car_coord_point_set_size =
+        lane_object.curve_car_coord_point_set.size();
     if (lane_object.pos_type == base::LaneLinePositionType::EGO_LEFT) {
       if (debug_level_ >= 2) {
         AINFO << "[GetEgoLane]LEFT_lane_objects[" << i
               << "].curve_image_point_set.size(): " << curve_image_point_size;
+        AINFO << "[GetEgoLane]LEFT_lane_objects[" << i
+              << "].curve_car_coord_point_set_size: "
+              << curve_car_coord_point_set_size;
       }
       if (curve_image_point_size < min_laneline_length_for_cipv_) {
         *b_left_valid = false;
       } else {
         *b_left_valid = true;
 
+        for (size_t j = 0; j < curve_car_coord_point_set_size; ++j) {
+          // ground_point
+          x = lane_object.curve_car_coord_point_set[j].x;
+          y = lane_object.curve_car_coord_point_set[j].y;
+          egolane_ground->left_line.line_point.emplace_back(Point2Df(x, y));
```

Comment:
`emplace_back(x, y)` would work.

Reply:
Thank you for the review! I am replacing them now.